python/ql/src/experimental/Functions/NamingConventionsFunctions.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Misnamed function
+ * @description A function name that begins with an uppercase letter decreases readability.
+ * @kind problem
+ * @problem.severity recommendation
+ * @id py/misnamed-function
+ * @tags maintainability
+ */
+
+import python
+
+from Function f, string first_char
+where
+  f.inSource() and
+  first_char = f.getName().prefix(1) and
+  not first_char = first_char.toLowerCase() and
```

Comment:
Same comment as above, except with `Function` instead of `Class` in the helper predicate.